pkg/specs/pods.go

```diff
@@ -252,6 +252,12 @@ func createPostgresContainers(cluster apiv1.Cluster, envConfig EnvConfig) []core
 
 	addManagerLoggingOptions(cluster, &containers[0])
 
+	// by default, the failureThreshold for liveness probe is 3, we do not want
+	// to change it if the user has not set the liveness probe timeout
+	if getLivenessProbeFailureThreshold(cluster.GetLivenessProbeTimeout()) != 3 {
```

Comment:
I'm a bit scared of this "3" as a magic number. And I wouldn't like to have it even as a constant.
What would happen if a new API changed it? We'll we remember that?

I think it would be best just to use a `*int32` instead of an `int32` for this new parameter, and do this only if the user requested a custom configuration.

Reply:
Thanks